app/views/layouts/_nav.html.haml

```diff
@@ -12,6 +12,9 @@
         = link_to "Konyhák", kitchens_path, class: navLinkClass
       %li.py-1
         = link_to "Friss jelentések", appliance_comments_path, class: navLinkClass
+      - if am_i_admin?
```

Comment:
We could check if the action of the link is permitted for the `current_user` and render the button based on that.
Currently we could something like this:
```suggestion
      - if UserPolicy.new(current_user, current_user).index?
```
But if we create a constructor for the `UserPolicy` with one argument we could use something like this:
```suggestion
      - if UserPolicy.new(current_user).index?
```
The constructor must accept both one and two arguments. One way to achive this is with a default argument.
The mentioned constructor in the `UserPolicy` for this to work:
```ruby
  def initialize(user, record = nil)
    super
  end
```